rhesus-monkeys/scripts/convert-samples.py

```diff
@@ -0,0 +1,235 @@
+import polars as pl
+import re
+from pathlib import Path
+
+# Convert files from xlsx to csv
+df06 = pl.read_excel("../data-raw/data06.xlsx")
+df06.write_csv('../data-raw/data06.csv')
+df15 = pl.read_excel("../data-raw/data15.xlsx")
+df15.write_csv('../data-raw/data15.csv')
+df16 = pl.read_excel("../data-raw/data16.xlsx")
+df16.write_csv('../data-raw/data16.csv')
```

Comment:
You can convert this into a `map()` to run on all of them (or loop, but we prefer `map()`).